permissions/helpers.py

```diff
@@ -119,6 +134,10 @@ def can_modify_application(user: AnyUser, application: Application) -> bool:
         return True
     if application.application_round.service_sector is None:
```

Comment:
The check for empty service sector would need to be after the general & unit permission check(s) but since the whole model is going to be dropped, this and the call to `can_manage_service_sectors_applications` can be removed as in `can_read_application`

Reply:
Removed the service sector checks here.